scripts/shared/nginx/cgi-bin.conf

```diff
@@ -0,0 +1,7 @@
+   location /cgi-bin/ {
+     gzip off;
+     root  /var/www/html;
```

Comment:
Was done by design because its a subset of an indent (include in parent file)
```
server {
    listen 80;
    listen [::]:80;
    include /etc/nginx/site-path-enabled/*.conf;
    root /var/www/html;
}
```

But i think thats redundant ill correct.